botfront/imports/startup/client/routes.jsx

```diff
@@ -33,6 +35,8 @@ import ChatDemo from '../../ui/layouts/chat.demo';
 import Index from '../../ui/components/index';
 import store from '../../ui/store/store';
 
+const MigrationStatus = new Mongo.Collection('migrationStatus');
```

Comment:
I think it is a bit weird to have a Collection declaration here in a jsx file, It would be cleaner to have it under botfront/imports/api like all the other collections. also all collection declaration have a part that deny updates from the client side e.g :
```
StoryGroups.deny({
    insert() {
        return true;
    },
    update() {
        return true;
    },
    remove() {
        return true;
    },
});
```

Reply:
because it's not really a collection. it's just that subscriptions need to go via a collection in Meteor...